build.yaml

```diff
@@ -14,7 +14,7 @@ settings:
   '#10': See the expand_version.py for all the quirks here
   core_version: 3.0.0-dev
   g_stands_for: gentle
```

Comment:
Yes, we were wondering. As well as https://github.com/grpc/grpc/blob/master/doc/g_stands_for.md I presume. Suggestions?